src/interactions/scrollInteraction.ts

```diff
@@ -0,0 +1,58 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Interaction {
+
+  export type ScrollCallback = (p: Point, deltaAmount: number) => any;
+
+  export class Scroll extends Interaction.AbstractInteraction {
+
+    /**
+     * The number of pixels occupied in a line.
+     */
+    public static PIXELS_PER_LINE = 120;
+
+    private _mouseDispatcher: Dispatcher.Mouse;
+    private _scrollCallback: ScrollCallback;
+
+    public _anchor(component: Component.AbstractComponent, hitBox: D3.Selection) {
+      super._anchor(component, hitBox);
+      this._mouseDispatcher = Dispatcher.Mouse.getDispatcher(<SVGElement> this._componentToListenTo.content().node());
+      this._mouseDispatcher.onWheel("Interaction.Scroll" + this.getID(), (p: Point, e: WheelEvent) => this._handleScrollEvent(p, e));
+    }
+
+    private _handleScrollEvent(p: Point, e: WheelEvent) {
+      var translatedP = this._translateToComponentSpace(p);
+      if (this._isInsideComponent(translatedP)) {
+        if (this._scrollCallback) {
+          e.preventDefault();
+          var deltaPixelAmount = e.deltaY * (e.deltaMode ? Scroll.PIXELS_PER_LINE : 1);
```

Comment:
While true, I've looked around implementations and only saw the 2 modes being cased out... I would need the PIXELS_PER_PAGE to switch off